### PR TITLE
More minor, less powerceep wizden reverts and changes

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About the PR
 Smaller version of #382
Ion storm Racism
Elite Hardsuit slowdown removed
Arti portals are 120 seconds again
Bananium is Radioactive again

## Why / Balance
good balance. 
## Technical details
Racism bit is a joke, it just allows for Ion laws to focus on a specific type of crewmember for those who cant read the code.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Hmeister
- tweak: Ion storms can now focus on specific species
- tweak: Elite Hardsuit nerf reverted
- tweak: Artifact portals last 120 seconds again
- tweak: Bananium is Radioactive again

### CODE REVIEW

**Comment by Hmeister-real** [comment]
no clue why it failed

**Review by Piras314 — CHANGES_REQUESTED** [review]
No ion storm racism.

**Comment by Hmeister-real** [comment]
> No ion storm racism.

But, but, i want borgs to be funny...

File: Resources/Prototypes/Datasets/ion_storm.yml [hunk 1]
```diff
@@ -837,6 +837,26 @@
...
+  - MOFFERS
+  - HARPYS
+  - VOX
+  - VULPKANIN
+  - GINGERBREAD PEOPLE
+  - KOBOLDS
+  - MONKEYS
+  - SLIME PEOPLE
+  - SKELETONS
+  - FURRIES # debating if i should add this, even though its funny
```
Comment (Aidenkrz) [inline Resources/Prototypes/Datasets/ion_storm.yml:858]: its funny but people will get butthurt

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 2]
```diff
@@ -564,7 +564,7 @@
     size: Huge
   - type: ClothingSpeedModifier
     walkModifier: 1.0
-    sprintModifier: 0.9
+    sprintModifier: 1.0
```
Comment (Aidenkrz) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:567]: still dont think i really agree with this, maybe make it 0.95

**Review by Aidenkrz — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml (hunk 2 above)
Comment (lanse12) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:567]: image

Comment (Hmeister-real) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:567]: they are the second like probably lol

File: Resources/Prototypes/Datasets/ion_storm.yml (hunk 1 above)
Comment (Hmeister-real) [reply Resources/Prototypes/Datasets/ion_storm.yml:858]: We shall see, if they do we can remove it

**Comment by Hmeister-real** [comment]
im actually going to stand for the Ion law race laws because they allow for funny outcomes like "All Harpys are now cool" or something. I want it test merged and if people complain il make a PR removing them

**Comment by Piras314** [comment]
People have complained about them, that's why they were removed from wizden, you can read the "Remove racism" PR on wizden to find out more. 

TLDR: It literally gives borgs laws that tell them to violate the rules.

**Review by misandrie — CHANGES_REQUESTED** [review]
Instead of pressing 4 buttons to revert try replicating tg/para's BHRS which allowed you to toggle between EVA (10% slowdown) and combat (no slowdown, no eva) modes.

Have  fun

**Comment by Hmeister-real** [comment]
> Instead of pressing 4 buttons to revert try replicating tg/para's BHRS which allowed you to toggle between EVA (10% slowdown) and combat (no slowdown, no eva) modes.
> 
> Have fun

idk how to actually code. I just do very basic stuff

**Comment by JoeHammad1844** [comment]
> im actually going to stand for the Ion law race laws because they allow for funny outcomes like "All Harpys are now cool" or something. I want it test merged and if people complain il make a PR removing them

agreed, nothing wrong with a little species based humor

**Comment by Piras314** [comment]
This isn't getting merged if you don't remove specism/racism

**Comment by misandrie** [comment]
> This isn't getting merged if you don't remove specism/racism

Changes generally are cring

**Comment by misandrie** [comment]
also guy presses "resolve conversation" himself after replying once lol lmao

**Comment by Verbalase** [comment]
> also guy presses "resolve conversation" himself after replying once and doing nothing lol lmao

which one is it, is it “lol”, or perhaps “lmao”

**anyway murderbone borg law!?!?!?!?!?**

**Comment by Aidenkrz** [comment]
no intention to make requested changes, closing for now
